scripts/codemods/overloads_no_missing.py

```diff
@@ -0,0 +1,49 @@
+from typing import List, Optional
+
+import libcst as cst
+import libcst.matchers as m
+from libcst import codemod
+
+EllipsisType = type(Ellipsis)
+
+
+class EllipsisOverloads(codemod.VisitorBasedCodemodCommand):
+    DESCRIPTION = "Ensure that `MISSING` is not used in any overloads as a default."
+
+    def visit_FunctionDef(self, node: cst.FunctionDef) -> Optional[bool]:
+        return False
+
+    def leave_FunctionDef(
+        self, node: cst.FunctionDef, new_node: cst.FunctionDef
+    ) -> cst.FunctionDef:
+        for deco in node.decorators:
+            if m.matches(deco.decorator, m.Name("overload")):
+                break
+        else:
+            return node
+
+        for param_type in ["params", "kwonly_params", "posonly_params"]:
+            params: List[cst.Param] = getattr(node.params, param_type, [])
```

Comment:
this should be typed as `Sequence` (they seem to be tuples by default)

Reply:
https://github.com/DisnakeDev/disnake/pull/708/commits/6712f1be4f0e6a9a110fb3c2bf829b8636ea4638